projects/igniteui-angular/src/lib/grids/grid/grid-row.component.html

```diff
@@ -1,3 +1,12 @@
+
+ <ng-container *ngTemplateOutlet='addRow ? addTemp : defaultTemp'></ng-container>
+
+<ng-template #addTemp>
+    <div class='igx-grid__tr--inner' [class.igx-grid__tr--add-animate]='animateAdd' (webkitAnimationEnd)='animationEndHandler()' (transitionend)='animationEndHandler()'>
```

Comment:
Maybe the  `transitionEnd` event can be omitted since both `webkitAnimationEnd` and `transitionend` events represent the same animation, because the animation itself contains some transitions.  Not sure which `webkitAnimationEnd` or `animationEnd` is supported by all browsers though. That needs checking.